ruby-challenges.rb

```diff
@@ -2,29 +2,100 @@
 # MINASWAN
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
-
+# Okay, I must take this array and return an array with the words that contain the letter of my choosing
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
 letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
-
-
+# the method must take two arguments
+def bev_array(array, ltr)
```

Comment:
Really great work in this section!  I appreciate that you added in your pseudocode for these problems as well.  I would recommend reading deeper into using attr_accessor in a Ruby class.  This eliminates the need to create your own getter and setters for those particular arguments which is super helpful.